examples/kafka_reader/kafka_reader.go

```diff
@@ -12,6 +12,7 @@ import (
 var (
 	brokers = flag.String("brokers", "127.0.0.1:9092", "a list of comma separated broker:port")
 	topic   = flag.String("topic", "", "the topic name")
+	group   = flag.String("group", "group", "the consumer group name")
```

Comment:
default group to filepath.Base(os.Args[0]) here, instead of the other two places.